lib/Segments/DynamicSegments/Filters/MailPoetCustomFields.php

```diff
@@ -124,6 +124,15 @@ private function applyEquality(QueryBuilder $queryBuilder, DynamicSegmentFilterE
     if ($operator === 'equals') {
       $queryBuilder->andWhere("subscribers_custom_field.value = $valueParam");
       $queryBuilder->setParameter($valueParam, $value);
+    } elseif ($operator === 'not_equals') {
+      $queryBuilder->andWhere("subscribers_custom_field.value != $valueParam");
+      $queryBuilder->setParameter($valueParam, $value);
+    } elseif ($operator === 'more_than') {
+      $queryBuilder->andWhere("subscribers_custom_field.value >= $valueParam");
```

Comment:
Shouldn't we rather use only more than (>) instead of more than or equal (=>) here? I think a user can create more than or equal by combining two conditions (more than and equal). Alternatively, they could enter a lower value, but I would say it is not a good UX. What do you think?